examples/staticcheck/WORKSPACE

```diff
@@ -0,0 +1,49 @@
+load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive")
+
+http_archive(
+    name = "com_github_sluongng_nogo_analyzer",
+    sha256 = "a74a5e44751d292d17bd879e5aa8b40baa94b5dc2f043df1e3acbb3e23ead073",
+    strip_prefix = "nogo-analyzer-0.0.2",
+    urls = [
+        "https://github.com/sluongng/nogo-analyzer/archive/refs/tags/v0.0.2.tar.gz",
+    ],
+)
```

Comment:
Please use `local_repo` instead to point back to the outer repo.

Do keep this in comment form though, as it will help folks understand why `local_repo` was used.

Reply:
Converted to `local_repository`.